### DISCUSSION TITLE
SFTP Test Write provides no response and throws [FileConnectorServlet Thread (Test Write) < qtp1763015813-85512] Server: Kerberos username [username]:

### DISCUSSION THREAD

**Discussion by aungktun** [nextgenhealthcare/connect#5887] [Q&A] [answered]
Trying to setup sftp writer in Mirth 3.5.0 and when testing using Test Write buttton no response back and throw this error in Dashboard 
"ERROR (Server:143): Kerberos username [username]" where username is the name of user running the mirth instance.

Sftp credential confirmed via WinSCP and it works.

Any idea what " Kerberos username [username]" error is ?

**Comment by rjkroll** [comment, marked as answer]
What O/S are you running and what versions of Mirth and Java are you using?   This may be related to the following issue:

https://github.com/nextgenhealthcare/connect/issues/3350

I've also seen threads where this issue has the resolution of some 3rd party library being used/called in the process somewhere.   In another thread, this same error was caused by a call to a 3rrd party library used for ZIPs.

**Reply by aungktun** [reply]
Thanks @rjkroll. It is working now.

**Reply by aungktun** [reply]
Why searching on word Kerberos not returning with #3350 ? 

This https://github.com/nextgenhealthcare/connect/issues/3350#issuecomment-626950527 solved the issue.
